tests/e2e/identity-flows.test.ts

```diff
@@ -0,0 +1,178 @@
+import {
+  openBrowser,
+  closeBrowser,
+  text,
+  click,
+  screenshot,
+  below,
+  waitFor,
+  write,
+  into,
+  textBox,
+  button,
+  checkBox,
+} from "taiko";
+
+import { authenticateToConnect, getCode, step } from "./utils";
+import * as locales from "@content/locales";
+
+describe("Identity flows", () => {
+  jest.setTimeout(750000);
+
+  beforeAll(async () => {
+    await openBrowser({
+      args: [
+        "--window-size=1440,1000",
+        "--no-sandbox",
+        "--start-maximized",
+        "--disable-dev-shm",
+      ],
+      headless: true,
+      observe: true,
+      observeTime: 500,
+    });
+  });
+
+  afterAll(async () => {
+    await closeBrowser();
+  });
+
+  const localizedAlertMessagesStrings = locales.en.alertMessages;
+  const localizedStrings = {
+    accountOverview: locales.en["/account"],
+    identitiesOverview: locales.en["/account/logins"],
+    identityOverview: locales.en["/account/logins/[type]/[id]"],
+    createIdentity: locales.en["/account/logins/[type]/new"],
+    updateIdentity: locales.en["/account/logins/[type]/[id]/update"],
+  };
+
+  it("Should be able to create a new identity", async () => {
+    try {
+      expect.assertions(6);
+
+      step("identities CRUD flows", true);
```

Comment:
This is the new function that prints what we have on the screenshot in the PR comment.